src/components/Template/Template.stories.tsx

```diff
@@ -247,14 +254,14 @@ export const TemplateFullExample: Story = {
             subHeaderText={otherSubHeaderText}
           />
         </TemplateBreakout>
-        <TemplateTop>
+        <TemplateFull>
           <Banner
             content="This is the top content area!"
             heading="Content Top"
             type="informative"
```

Comment:
Do we still want to use this `content` or `heading`? Feels a little like relics of the old component now. We could swap it for the values you used in the "Complex Example":

```
<TemplateFull>
          <Banner
            content="This is a full content area!"
            heading="Full content"
            type="informative"
          />
        </TemplateFull>
```
If you decide to update, then same goes for the "Content Bottom" version and those in other stories.